submissions/retrospective/conor-retrospective.md

```diff
@@ -0,0 +1,11 @@
+When I first heard about Alt-F4 being started, I immediately loved the idea and knew I had to get involved. For context, at the time I had very little to due to a/the pandemic and was desperate for a long term project to work on. I was searching high and low for anything that I could spend my time on when I found Alt-F4, I knew immediately it was the project for me. There was just the minor issue that I had no idea what I could contribute to the project. For me writing has always been a bit of a slog and I couldn’t think of any particularly good ideas for things to write about anyway so I threw my hat into the Techie ring. 
```

Comment:
missing word? "I had very little" [TIME] ?

Reply:
Yep meant to be "very little to do" will fix